state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -1894,25 +1863,37 @@ impl<T: Config> Pallet<T> {
 		pool: &mut Pool<T>,
 		asset_pair: &AssetPair,
 		lp: &T::AccountId,
-		side: Side,
-		order: OrderId,
+		order: Order,
```

Comment:
nit: I think Side was a better name. Order suggests it's the actual order. (but I can see `Side` already has another meaning). 

How about OrderSide or BuyOrSell or something less ambiguous?

Reply:
OrderType?

Reply:
Order types are generally used to refer to things like limit order / market order / fill or kill etc. Ie. it's a collection of attributes or criteria.